extensions/wikia/AdEngine/js/run/mercury.run.js

```diff
@@ -16,12 +16,12 @@ require([
 	'wikia.window'
 ], function (
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 14 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)